src/logion/model/locrequest.model.ts

```diff
@@ -190,7 +197,7 @@ export class LocRequestAggregateRoot {
     }
 
     addMetadataItem(itemDescription: MetadataItemDescription) {
-        this.ensureOpen();
+        this.ensureOpen(true);
```

Comment:
This should fail because called by controller, no?

Reply:
It's called only during synchro.